assets/css/main.css

```diff
@@ -14,3 +14,5 @@
 @import "sections/form";
 
 @import "pages/home";
+
+@import "tasks/_contact-tasks"
```

Comment:
It's good that you remembered to add this import without the .css extension :)
There is one thing to change. Remove lodash from "contact tasks". Then everything should work fine. We add css file names without extension and lodash, because our server needs this format.